### ISSUE TITLE
Create event models

### ISSUE THREAD

**Issue by ronaldtse** [riboseinc/kaohi#6]
Based on https://github.com/riboseinc/event-configuration-models.

**Comment by ronaldtse** [comment]
Types of events:

Log source events (i.e. log file):
* LogSourceCreate
* LogSourceDelete
* LogSourceOpen
* LogSourceClose
* LogSourceRotate
* LogSourceLost / Overwritten
* LogSourceModify

Daemon events:
* DaemonStart
* DaemonConfigChange
** Needs more detailed configuration options
* DaemonStop

Rsyslog related:
* RsyslogDaemonStart
** port, etc.
* RsyslogDaemonStop
* RsyslogReceiveMessage
** datetime, from host, service, etc.

Thoughts @erikbor ?

**Comment by jjr840430** [comment]
@ronaldtse Do I need to monitor the creation and deletion of log files?
The first requirement is just to monitor the changes of log file like as `tail -f` mode

**Comment by erikbor** [comment]
@jjr840430 yes, so also means that file descriptors need to be verified with the path that was defined
